pkg/proxy/conntrack/cleanup.go

```diff
@@ -52,7 +60,17 @@ func CleanStaleEntries(ct Interface, ipFamily v1.IPFamily,
 		}
 	}
 
-	// serviceIPEndpointIPs maps service IPs (ClusterIP, LoadBalancerIPs and ExternalIPs)
+	// filter UDP entries only (in-place https://go.dev/wiki/SliceTricks)
```

Comment:
improvement, we filter UDP entries only once, before we were filtering in each iteration